firebase-dev.json

```diff
@@ -0,0 +1,23 @@
+{
+  "hosting":
+    {
+      "site": "namerequest-dev",
+      "public": "dist",
+      "ignore": ["**/devops/**", "**/.*", "**/node_modules/**"],
+      "headers" : [ {
+        "source": "**",
+        "headers" : [
+          { "key" : "Access-Control-Allow-Origin", "value" : "*" },
+          { "key" : "X-Frame-Options", "value" : "DENY" },
+          { "key" : "X-Content-Type-Options", "value" : "nosniff" },
+          { "key" : "X-XSS-Protection", "value" : "1; mode=block" },
+          {
+            "key": "Content-Security-Policy",
+            "value": "default-src 'self'; frame-src 'self' *.gov.bc.ca *.hotjar.com; script-src 'self' 'unsafe-eval' 'unsafe-inline' *.gov.bc.ca *.hotjar.com *.googleapis.com; style-src 'self' 'unsafe-inline' *.cloudflare.com *.googleapis.com *.jsdelivr.net; font-src 'self' *.gov.bc.ca *.hotjar.com *.cloudflare.com *.gstatic.com *.jsdelivr.net; img-src 'self' *.hotjar.com *.postescanada-canadapost.ca data:; connect-src 'self' *.gov.bc.ca *.launchdarkly.com *.hotjar.com *.postescanada-canadapost.ca *.sentry.io *.apigee.net wss://*.hotjar.com *.hotjar.io; manifest-src 'self';"},
```

Comment:
Did you take this from the existing nginx config? Asking because Karim recently had to update it.